pkg/kubelet/rkt/rkt.go

```diff
@@ -1459,35 +1417,24 @@ func (r *Runtime) getPodInfo(uuid string) (*podInfo, error) {
 	return info, nil
 }
 
-// getImageByName tries to find the image info with the given image name.
-// TODO(yifan): Replace with 'rkt image cat-manifest'.
-// imageName should be in the form of 'example.com/app:latest', which should matches
-// the result of 'rkt image list'. If the version is empty, then 'latest' is assumed.
-func (r *Runtime) getImageByName(imageName string) (*kubecontainer.Image, error) {
-	// TODO(yifan): Print hash in 'rkt image cat-manifest'?
-	images, err := r.ListImages()
+// buildImageName constructs the image name for kubecontainer.Image.
+func buildImageName(img *rktapi.Image) string {
+	return fmt.Sprintf("%s:%s", img.Name, img.Version)
```

Comment:
@jonboulle The result returned by api service will always contain the version. "latest" by default.